### DISCUSSION TITLE
Show Collection of Tags after every post

### DISCUSSION THREAD

**Discussion by mariolambe** [GetPublii/Publii#1529] [Help]
I use the theme Sources and I would like to know if it is possible to show after every post the list of tags.
<img width="1359" alt="Screenshot 2024-06-02 at 13 06 33" src="https://github.com/GetPublii/Publii/assets/61537859/5abf2561-dee7-4beb-9a95-20ab19d13261">

Thanks for any support!

**Comment by 3muchfun** [comment]
Hi, for my blog I solved it this way with some programming: I opened the post.hbs and added the following:

```
<p class="post-categories">
                    <span>Categories:</span>
                    {{#checkIfAny @config.post.displayTags }}
                        {{#if @config.post.displayTags}}
                            {{#if tags}}
                                {{#each tags}}
                                    {{#checkIf @index '>' 0}}, {{/checkIf}}
                                    <a href="{{url}}">{{name}}</a>
                                {{/each}}
                            {{/if}}
                        {{/if}}
                    {{/checkIfAny}}
                </p>
```

I think you need to and some HTML to achieve your screenshot
